eventsourcing/src/main/java/org/axonframework/eventsourcing/AbstractSnapshotter.java

```diff
@@ -205,6 +264,32 @@ public void run() {
                 }
             }
         }
+
+        public Runnable andFinally(Runnable r) {
```

Comment:
I think this method should be `private`.

Reply:
I disagree. This method is intended to be invoked from outside this class. It happens to be an inner class right now, but there is bo real reason why not to make this a public utility class. In that case, a private method wouldn't work.

Reply:
Through GitHub code folding magic, I completely missed this method was part of the `SilentTask` class...mea culpa.

I agree with you @abuijze, this should stay `public`, thanks for correcting me.